### PR TITLE
Gitops issue 201- Changes in bootstrap command

### PR DESCRIPTION
**Pull request by ishitasequeira**
**What type of PR is this?**
 /kind design

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-201

**How to test changes / Special notes to the reviewer**:
Acceptance Criteria:

    Rename the flag  app-repo-url to service-repo-url and app-webhook-secret to service-webhook-secret
    The app name should be "app-<repo-name>"
    The service name should be "repo-name"

Co-authored by @gaganhegde and @ishitasequeira

### CODE REVIEW

File: pkg/odo/cli/pipelines/bootstrap.go [hunk 1]
```diff
@@ -97,8 +97,8 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 	initCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
 	initCmd.Flags().StringVar(&o.GitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub webhook secret for GitOps repository (if not provided, it will be auto-generated)")
 
-	initCmd.Flags().StringVar(&o.AppRepoURL, "app-repo-url", "", "Application source e.g. https://github.com/organisation/application")
-	initCmd.Flags().StringVar(&o.AppWebhookSecret, "app-webhook-secret", "", "Provide the GitHub webhook secret for Application repository (if not provided, it will be auto-generated)")
+	initCmd.Flags().StringVar(&o.ServiceRepoURL, "service-repo-url", "", "Application source e.g. https://github.com/organisation/application")
```
Comment (bigkevmcd) [inline pkg/odo/cli/pipelines/bootstrap.go]: Shouldn't this say "Service source" and reference a "service" url?

File: pkg/pipelines/bootstrap.go [hunk 2]
```diff
@@ -35,8 +35,8 @@ const (
 type BootstrapOptions struct {
 	GitOpsRepoURL            string // This is where the pipelines and configuration are.
 	GitOpsWebhookSecret      string // This is the secret for authenticating hooks from your GitOps repo.
-	AppRepoURL               string // This is the full URL to your GitHub repository for your app source.
-	AppWebhookSecret         string // This is the secret for authenticating hooks from your app source.
+	ServiceRepoURL           string // This is the full URL to your GitHub repository for your app source.
+	ServiceWebhookSecret     string // This is the secret for authenticating hooks from your app source.
```
Comment (bigkevmcd) [inline pkg/pipelines/bootstrap.go]: Same here?

**Review by bigkevmcd — CHANGES_REQUESTED** [review]
Looks pretty good, some minor textual changes.

File: pkg/odo/cli/pipelines/bootstrap.go (hunk 1 above)
Comment (ishitasequeira) [reply pkg/odo/cli/pipelines/bootstrap.go]: made the requested changed

File: pkg/pipelines/bootstrap.go (hunk 2 above)
Comment (ishitasequeira) [reply pkg/pipelines/bootstrap.go]: made the requested changed

**Review by bigkevmcd — APPROVED** [review]

**Pull request merged**